conda_build/build.py

```diff
@@ -592,7 +592,7 @@ def test(m, move_broken=True):
     specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
 
     # add packages listed in test/requires
-    specs += m.get_value('test/requires', [])
+    specs.extend(ms.spec for ms in m.ms_depends('run'))
 
```

Comment:
Since it's here, just want to get it off my chest...

I've never understood the incongruency/asymmetry between `requirements/build` and `test/requires`. 

Ok, I feel better now. 

> On Apr 28, 2016, at 12:59 AM, Michael C. Grant <notifications@github.com> wrote:
> 
> In conda_build/build.py:
> 
> > @@ -592,7 +592,7 @@ def test(m, move_broken=True):
> >      specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
> >  
> >      # add packages listed in test/requires
> > -    specs += m.get_value('test/requires', [])
> > +    specs.extend(ms.spec for ms in m.ms_depends('run'))
> >  
> Oh, good point, it should be both, shouldn't it.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>